_Sidebar.md

```diff
@@ -127,6 +127,7 @@
   * [[Mobile device testing|Mobile-device-testing]]
   * [[Performance testing|Performance-Testing]]
   * [[Build process|Build-process]]
+  * [[Partial CI Tests Structure|Partial-CI-Tests-Structure]]
```

Comment:
Should this go under some set of pages that relate to testing?

Reply:
Done, put it under Testing.